tests/lib/SimpleSAML/Utils/AttributesTest.php

```diff
@@ -76,10 +75,8 @@ public function testGetExpectedAttributeMissingAttribute()
             'attribute' => ['value'],
         ];
         $expected = 'missing';
-        $this->expectException(
-            \SimpleSAML\Error\Exception::class,
-            "No such attribute '".$expected."' found."
-        );
+        $this->expectException(\SimpleSAML\Error\Exception::class);
+        $this->expectExceptionMessage("No such attribute '".$expected."' found.");
         Attributes::getExpectedAttribute($attributes, $expected);
```

Comment:
Not sure in what PHPunit-version this was allowed, but today it's split in two calls (actually for years already)